src/components/calculator/PrevalenceControls.tsx

```diff
@@ -155,6 +175,22 @@ export const PrevalenceControls: React.FunctionComponent<{
     (option) => option.value === data.subLocation,
   )
 
+  const subSubLocationOptions = !showSubSubLocation
+    ? []
+    : Locations[data.subLocation].subdivisions.map((locKey) => {
+        return { label: Locations[locKey].label, value: locKey }
+      })
+  console.log(
+    'show non us counties:',
+    showSubSubLocation,
+    data.subLocation,
+    'options:',
+    subSubLocationOptions,
+  )
```

Comment:
Do you still want this in the code when we commit?

Reply:
nah.